### ISSUE TITLE
Project Comment: Marion Riley

### ISSUE THREAD

**Issue by marionbriley** [crissyshan/Soviet-Rock-Songs#18]
Hi! I don't know if I've looked at your repo before, but I just want to comment first of all that I love how you are divying up tasks and using the to-do list! It seems like a very organized system. Also, for my actual substantive comment, I saw you all were debating only including analysis of songs rather than both the analysis and the lyrics. I think, as someone who knows very little Soviet rock (read: none), I think it would be beneficial for users to have easy access to lyrics. If you're worried about making the page too busy, you could maybe have a link with your analysis to a page with the lyrics, or some sort of button where you can toggle the lyrics to appear or disappear. Just my thoughts:) Keep up the good work!

**Comment by dap167** [comment]
I definitely concur with this. Personally, it seems strange to not include the songs at all. One thing that you could possibly do might be similar to the XSLT Assignment #3 where we applied CSS to `<span>` elements so that they could show up differently in the page. This might be a pretty good way to show off the lyrics and markup with the corresponding graphs

**Comment by mjyb16** [comment]
Yes, these are good points; we are still seriously considering including the lyrics in the website, and the current mock-up of the website (soon to be put online) does include a lyrics tab. We will probably convert our analytical markup to a link that opens a popup with the relevant markup described.
